ghcloneall.py

```diff
@@ -441,6 +450,7 @@ def list_repos(self, user=None, organization=None, pattern=None,
                         '?sort=full_name').format(
                             owner)
         elif user and not organization:
+            self._verify_user_token(user)
```

Comment:
I think this should be moved inside the `if include_private and self.has_auth_token:`

Reply:
Fixed